### ISSUE TITLE
CNCF SIG-Runtime Discussion/Presentation?

### ISSUE THREAD

**Issue by raravena80** [paritytech/wasmi#253]
Hello wasmi team,

I'm one of the co-chairs of the CNCF SIG-Runtime, I'm reaching out and think it would be great for you to present/discuss the project at one of our meetings. An overview of the project would be great.

Let me know if this something you'd be interested in doing. If yes, please feel free to add it to our [agenda](https://bit.ly/cncf-sig-runtime-meeting-notes) or reach out to me (raravena80 at gmail.com)

Thanks!

**Comment by Robbepop** [comment]
@raravena80 Sorry for the late reply. What information do you guys need about `wasmi` and is this still needed?

**Comment by raravena80** [comment]
@Robbepop np. It would be great to have you present in our meeting.  A general overview of the project, its community and how it would fit cloud native applications. We would also be interested to hear how it's used in blockchain apps. How about Jan 20th or Feb 3rd? Thanks!

**Comment by Robbepop** [comment]
I am no expert in cloud computing myself so I'd need to know what the requirements of a Cloud are with respect to interpreters such as `wasmi` before I can provide more details about applicability.

**Comment by raravena80** [comment]
Can `wasmi` be used to test WASM modules? It's an interpreter right? i.e dry run (or prototype) before running the module in a production environment.    Additionally, are there any plans to support or provide a path to [WASI](https://wasi.dev/)?

**Comment by Robbepop** [comment]
Yes, `wasmi` is a Wasm interpreter and is able to execute Wasm modules. I am not entirely sure what exactly you mean with "dry running" but form what I understood the host side should be able to determine how the Wasm module executes and how state from the execution is synced or not.
We do not plan to implement WASI out of the box, however, it is very well possible for users of `wasmi` to provide their own WASI support using the existing `wasmi`.

**Comment by raravena80** [comment]
In any case it would be good to hear a general overview of the project and a demo.  Will that work?

**Comment by Robbepop** [comment]
At this point I cannot give a guarantee. Right now we are working on a big update for `wasmi` that will take some more time. It won't make sense to talk a lot about `wasmi` until that's done. I will get back to you once we are ready.

**Comment by raravena80** [comment]
Sounds good, thanks!

**Comment by Robbepop** [comment]
https://github.com/paritytech/wasmi/issues/358 is probably important for this. Can you confirm this @raravena80 ?

**Comment by raravena80** [comment]
Yes, correct.  Thanks!
